runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CrypticClue.java

```diff
@@ -2294,7 +2294,7 @@ public Collection<Integer> getNpcRegions()
 	@Override
 	public int[] getConfigKeys()
 	{
-		return new int[]{text.hashCode()};
+		return new int[]{text.hashCode() + itemId};
```

Comment:
This was changed due to duplicate Sherlock steps. Interested in a better way to handle those if this is not sufficient.